airflow/models/taskinstance.py

```diff
@@ -1871,3 +1872,15 @@ def construct_task_instance(self, session=None, lock_for_update=False) -> TaskIn
         else:
             ti = qry.first()
         return ti
+
+
+STATICA_HACK = True
+globals()['kcah_acitats'[::-1].upper()] = False
+if STATICA_HACK:  # pragma: no cover
+    # Let pylint know about these relationships, without introducing an import cycle
+    from sqlalchemy.orm import relationship
+
+    from ..job import base_job
+    from . import dagrun
```

Comment:
I think we usually try to avoid dot imports in Airflow, any reason to use them here?

Reply:
No good reason, no. Laziness on my part.

I think we avoid them in Airflow because of Py2 oddness with them, and stop we've never reconsidered since dropping support for Py2 (at least not that I've seen)

But also other than shorter lines there's no functional difference so :shrug: